app/src/main/java/com/example/hangouts/BayesianRating.java

```diff
@@ -0,0 +1,37 @@
+package com.example.hangouts;
+
+import org.apache.commons.math3.distribution.NormalDistribution;
+
+import java.util.List;
+
+public class BayesianRating {
+
+    final static int standardNormalMean = 0;
+    final static int standardStdDeviation = 1;
+    final static NormalDistribution standardNormalDist = new NormalDistribution(standardNormalMean,
+            standardStdDeviation);
+
+    private static double getPpf(double x) {
+        return standardNormalDist.inverseCumulativeProbability(x);
+    }
+
+    public static double getBayesianRating(List<Double> list, double significance) {
```

Comment:
a javadoc comment on the class and this method would be very helpful.  Every one that comes here will not know what a BayesianRating is, or what it can be used for.

Same for getPpf - what is ppf? is it worth having a separate method if it is just calling a single method?